README.md

```diff
@@ -1,16 +1,80 @@
 # ansible-role-packer-binary
 
-A brief description of the role goes here.
+Install `packer` binary from packer.io
+
+## Note for CentOS users
+
+The default installation of CentOS has `/sbin/packer`, linked to
+`cracklib-packer`. The `packer` from packer.io is renamed to `packer.io`.
 
 # Requirements
 
 None
 
 # Role Variables
 
-| variable | description | default |
+| Variable | Description | Default |
 |----------|-------------|---------|
-
+| `packer_binary_version` | version of `packer` | `1.0.4` |
+| `packer_binary_base_url` | base URL of remote file | `https://releases.hashicorp.com/packer/{{ packer_binary_version }}` |
+| `packer_binary_os_type` | type of OS | `{{ __packer_binary_os_type }}` |
+| `packer_binary_arch` | architecture | `amd64` |
+| `packer_binary_filename` | remote file name | `packer_{{ packer_binary_version }}_{{ packer_binary_os_type }}_{{ packer_binary_arch }}.zip` |
+| `packer_binary_sym_name` | remote file name without extension | `{{ packer_binary_filename | splitext | first }}` |
+| `packer_binary_url` | URL of remote file | `{{ packer_binary_base_url }}/{{ packer_binary_filename }}` |
+| `packer_binary_dir` | path to directory to install `packer` | `{{ __packer_binary_dir }}` |
+| `packer_binary_name` | file name of symlink to `packer` binary | `{{ __packer_binary_name }}` |
+| `packer_binary_path` | path to symlink | `{{ packer_binary_dir }}/{{ packer_binary_name }}` |
+| `packer_binary_owner` | owner of files and directories | `{{ __packer_binary_owner }}` |
+| `packer_binary_group` | group of files and directories| `{{ __packer_binary_group }}` |
+| `packer_binary_mode` | mode of `packer` binary | `755` |
+| `packer_binary_cache_dir` | path to directory where archives are kept | `{{ __packer_binary_cache_dir }}` |
+| `packer_binary_sha256sum` | list of SHA 256 sum of remote files | `{"packer_1.0.4_freebsd_amd64.zip"=>"8c6aebcaf228d6883cd6afda4b88028cd8168fd04d92feb016b7b4a79571d0a2", "packer_1.0.4_linux_amd64.zip"=>"646da085cbcb8c666474d500a44d933df533cf4f1ff286193d67b51372c3c59e", "packer_1.0.4_openbsd_amd64.zip"=>"487ca4f29c327b389370768cb303ade4d791854918d52632be305ac9e614e64f"}` |
```

Comment:
do you mean that you are planning to add the other versions to the dict so that users can choose the version?